### PR TITLE
MMCA-3664 Update help and support changes on GGA

### PR DESCRIPTION
**Pull request by bsgangapat**

### CODE REVIEW

File: app/views/guarantee_account.scala.html [hunk 1]
```diff
@@ -27,6 +27,7 @@
 h2: components.h2,
 p: components.p,
 link: components.link,
+newTabLink: components.newTabLink,
```
Comment (wg-hmrc) [inline app/views/guarantee_account.scala.html:30]: can not see where you are using `newTabLink`

File: app/views/components/link.scala.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
         pClass: String = "govuk-body")(implicit messages: Messages)
 
 @link = {
- <a @{linkId.fold("")(id => s"id=$id")} class="@linkClass" href="@location">@messages(linkMessage)</a>@if(linkSentence){.}
+ <a @{linkId.fold("")(id => s"id=$id")} target = "_blank" class="@linkClass" href="@location">@messages(linkMessage)</a>@if(linkSentence){.}
```
Comment (wg-hmrc) [inline app/views/components/link.scala.html:31]: won't this not open links using this component in a new tab ?  I would probably wrap `target = "_blank"` in a condition

**Review by sameerhmrc — APPROVED** [review]
Approved

File: app/views/guarantee_account.scala.html [hunk 3]
```diff
@@ -58,4 +58,15 @@
         pClass = "govuk-body govuk-!-padding-bottom-7",
         postLinkMessage = Some("cf.guarantee-account.transactions.request.link.post")
     )
+
+
+    @h2("help.heading")
+
+    @link(
+        preLinkMessage= Some("cf.guarantee-account.transactions.request.link.text"),
+        location = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/general-guarantee-accounts",
```
Comment (wg-hmrc) [inline app/views/guarantee_account.scala.html:67]: should we hardcode absolute links ?

**Pull request merged**